base/shell/explorer/taskswnd.cpp

```diff
@@ -315,17 +366,27 @@ class CTaskSwitchWnd :
     HTHEME m_Theme;
     UINT m_ButtonsPerLine;
     WORD m_ButtonCount;
+    WORD m_DefaultButtonCount;
 
     HIMAGELIST m_ImageList;
 
     BOOL m_IsGroupingEnabled;
     BOOL m_IsDestroying;
 
+    BOOL m_CloseTaskGroupOpen;
+
     SIZE m_ButtonSize;
 
+    HIMAGELIST m_TaskGroupImageList;
+
     UINT m_uHardErrorMsg;
     CHardErrorThread m_HardErrorThread;
 
+    INT TaskGroupOpened = -1;
```

Comment:
Could you group _all_ the variables pertaining to taskbar grouping support together?

Reply:
Should be grouped now